src/maxdiffusion/eval.py

```diff
@@ -74,8 +87,7 @@ def eval_scores(config, images_directory=None):
     captions_df = load_captions(config.caption_coco_file)
```

Comment:
It might be more prudent to parse the caption_coco_file for the ids and match the images and captions with that id, instead of looping over both lists and hoping they are both in order.

Reply:
I'll merge this revert in and see if I can quickly play around with this.

Reply:
@entrpn Ah see https://github.com/google/maxdiffusion/blob/a8b7bf018a4085aae84ac452dd5105d57f62bf73/src/maxdiffusion/eval.py#L71. That's already what's happening - `load_images` loads the images and then queries the captions_df for the corresponding prompt based on the prompt id. The returned lists are in order already. 

Let me know if you encounter the same issues - i can find some time to debug this further.

Reply:
Sounds good, thank you!